api/v1alpha1/application_types.go

```diff
@@ -127,19 +127,6 @@ type AuthorizationSettings struct {
 	AllowList []string `json:"allowList,omitempty"`
 }
 
-// ResourceRequirements
-// +kubebuilder:object:generate=true
-type ResourceRequirements struct {
```

Comment:
I assume this was no longer in use?

Reply:
It was not removed in the last PR, but should've been. It's also defined in the podtypes folder